pkg/http/core/artifacts_test.go

```diff
@@ -470,6 +470,59 @@ var _ = Describe("Artifacts", func() {
 			})
 		})
 
+		Context("when the artifact is type http/file", func() {
+			BeforeEach(func() {
+				body.Write([]byte(fmt.Sprintf(payloadRequestFetchHTTPFileArtifact, fakeFileServer.URL())))
+				createRequest(http.MethodPut)
+			})
+
+			When("getting the client returns an error", func() {
+				BeforeEach(func() {
+					fakeArtifactCredentialsController.HTTPClientForAccountNameReturns(nil, errors.New("error getting git client"))
```

Comment:
It's just a fake response in a test, but for clarity sake, the error message probably should be 
```
error getting http client
```